src/reactviews/pages/QueryResult/table/table.ts

```diff
@@ -245,10 +247,34 @@ export class Table<T extends Slick.SlickData> implements IThemable {
                             }
                         },
                     );
+                    let columnSortDirection =
+                        columnFilterMap[column.id!][0].sorted;
+                    if (
+                        (columnSortDirection === "ASC" ||
+                            columnSortDirection === "DESC") &&
+                        !sortDirection
+                    ) {
+                        sortColumn = column;
+                        (column as FilterableColumn<T>).sorted =
+                            columnSortDirection;
+                        sortDirection =
+                            columnSortDirection === "ASC" ? true : false;
+                    }
                 }
             }
         }
         await this._data.filter(this.columns);
+        if (sortDirection !== undefined && sortColumn) {
+            let sortArgs = {
+                grid: this._grid,
+                multiColumnSort: false,
+                sortCol: sortColumn,
+                sortAsc: sortDirection,
+            };
+            await this._data.sort(sortArgs);
+        }
+        //TODO: need to set sort button image to indicate sort is applied
```

Comment:
Is this something that you need to do for this current PR or a future one?

Reply:
already done, forgot to remove this